OpenSim/Simulation/Model/ContactGeometry.h

```diff
@@ -46,7 +46,7 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(ContactGeometry, ModelComponent);
 // DATA
 //=============================================================================
 protected:
-    Body* _body;
+    SimTK::ReferencePtr<PhysicalFrame> _body;
```

Comment:
I am not quite sure. I believe  that `const ReferencePtr<PhysicalFrame>` means the ReferencePtr member is const so that the ReferencePtr member cannot be changed with out a const_cast. In the case of `ReferencePtr<const PhysicalFrame> rp`  rp is modifiable but the type of pointer it contains is const (that is we cannot muck around with the PhyiscalFrame it points to), which is what I wanted.